sbin/cubeit

```diff
@@ -173,6 +178,94 @@ if [ -n "$ARTIFACTS_DIR" ]; then
 fi
 export ARTIFACTS_DIR
 
+# determine the location of key store
+if [ -z "$KEYS_DIR" -o ! -d "$KEYS_DIR" ]; then
+    SAMPLE_KEYS_DIR="$ARTIFACTS_DIR/sample-keys"
+    USER_KEYS_DIR="$ARTIFACTS_DIR/user-keys"
+
+    if [ -d "$USER_KEYS_DIR" ]; then
+        KEYS_DIR="$USER_KEYS_DIR"
+    else
+        KEYS_DIR="$SAMPLE_KEYS_DIR"
+    fi
+fi
+
+# check if the filesystem can support xattr.
+check_fs_for_ima()
+{
+    local tmpfile="`mktemp test_XXXX`"
+    local res=`setfattr -x security.ima "$tmpfile" 2>&1 | grep -q 'Operation not supported$'`
+
+    rm -f "$tmpfile"
+
+    return ! $res
+}
+
+if [ $DO_IMA_SIGN -eq 1 ]; then
```

Comment:
All of the new commits to the installer use ${var} .. that format should be maintained here. I haven't converted the entire file, simply because it would be too much noise, but eventually they will all be converted to that format.